server/config.py

```diff
@@ -0,0 +1,4 @@
+SQLALCHEMY_DATABASE_URI = "postgresql://localhost"
```

Comment:
We might consider checking this in if we start to store secret keys and other scary things in here. The alternative would be to have something like a `~/.bettermintrc` that developers have to source into their virtualenv.